core-web/libs/sdk/react/src/lib/components/DotcmsLayout/DotcmsLayout.tsx

```diff
@@ -38,6 +40,12 @@ export type DotcmsPageProps = {
 export function DotcmsLayout({ entity, config }: DotcmsPageProps): JSX.Element {
     const isInsideEditor = useDotcmsEditor(config);
 
+    const { shouldWaitForVariant } = useExperimentVariant(entity);
+
+    if (shouldWaitForVariant()) {
+        return <div></div>;
+    }
```

Comment:
If this is the inly solution in the world... don't added here in the SDK, you can either:

1. Move it to the example itself
2. Create a new `DotcmsLayoutExperiments` component.

Becuase this component only functionality should be render a dotCMS page and nothing more.

Reply:
Fixing using the option 2.